docker-compose.yaml

```diff
@@ -30,3 +30,31 @@ services:
         cd processor
         cargo build
       "
+
+  postgres:
+    image: postgres:alpine
+    container_name: bond-postgres
+    restart: always
+    environment:
+      POSTGRES_USER: sawtooth
+      POSTGRES_PASSWORD: sawtooth
+      POSTGRES_DB: sawtooth-bond
+    ports:
+      - '5432:5432'
+
+  sawtooth-bond-subscriber:
```

Comment:
Recommend changing this to `bond-subscriber` to match other services

Reply:
Updated